registry/handlers/app_test.go

```diff
@@ -26,7 +26,7 @@ import (
 func TestAppDispatcher(t *testing.T) {
 	driver := testdriver.New()
 	ctx := context.Background()
-	registry, err := storage.NewRegistry(ctx, driver, storage.BlobDescriptorCacheProvider(memorycache.NewInMemoryBlobDescriptorCacheProvider()), storage.EnableDelete, storage.EnableRedirect)
+	registry, err := storage.NewRegistry(ctx, driver, storage.BlobDescriptorCacheProvider(memorycache.NewInMemoryBlobDescriptorCacheProvider(0)), storage.EnableDelete, storage.EnableRedirect)
```

Comment:
Perhaps the same for the `0` (which means "unlimited?")

Reply:
Done